simpletodo/src/Card.css

```diff
@@ -0,0 +1,50 @@
+
+.card {
+    display: block;
+    flex-wrap: wrap;
+   /* width: 20%;*/
+    width: 330px;
+    margin-right: 40px;
+    margin-bottom: 40px;
+    justify-content: space-between;
+    border-color: black;
+    border-radius: 30px;
+    box-shadow: 0 4px 8px 0 rgba(17, 15, 15, 0.8);
+}
+
+.cardButton {
```

Comment:
Ezeknek az attribútumoknak a nagyrésze felül lesz írva a Mui-Button miatt, azokat vagy jobb kitörölni, vagy pedig azokat is `!important`-ozni ha szükséged van rájuk